teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -465,13 +475,15 @@ interface Build {
 interface Investigation {
     val id: InvestigationId
     val state: InvestigationState
-    val assigneeUsername: String
-    val reporterUsername: String?
+    val assignee: User
+    val reporter: User?
     val comment: String
     val resolveMethod: InvestigationResolveMethod
     val targetType: InvestigationTargetType
     val testIds: List<TestId>?
     val problemIds: List<BuildProblemId>?
+    val assignmentProject: Project?
```

Comment:
It looks confusing to have two properties `assignmentProject` and `assignmentBuildConfiguration` here. May it happen that the both contain non-null value? Probably we can replace them by a single `investigationScope` property?